corehq/apps/reports/tasks.py

```diff
@@ -337,11 +338,33 @@ def extract_form_info(form, properties=None, case_ids=case_ids):
 
         return form_info
 
-    key = [domain, app_id, xmlns]
-    form_ids = {f['id'] for f in XFormInstance.get_db().view("attachments/attachments",
-                                                             start_key=key + [startdate],
-                                                             end_key=key + [enddate, {}],
-                                                             reduce=False)}
+    if not export_is_legacy:
+        query = (FormES()
+                 .domain(domain)
+                 .app(app_id)
+                 .xmlns(xmlns)
+                 .submitted(gte=parse(startdate), lte=parse(enddate))
+                 .remove_default_filter("has_user"))
+        form_ids = set()
+        for form in query.scroll():
+            try:
+                for attachment in form['_attachments'].values():
```

Comment:
@millerdev no, they aren't a special class. we just have a generic wrapper for handling documents returned from ES (not specific to any model). reindex is semi painful, but doable. it requires setting up a new directory with hq on hqdb0 and babysitting a preindex for approximately 2 days. also not wild about the transform function, but could be a stop gap until we decide to do a full reindex? we'd just have to remember to change it. don't have too many strong opinions either way. just think it's ripe for error if we leave attachments in separate properties